File: dayTwo/python/level two/task_five.py
number_one = int(input("Enter a number: "))
number_two = int(input("Enter a number: "))
number_three = int(input("Enter a number: "))

largest = number_one

if number_two > largest:
    largest = number_two
elif number_three > largest:
    largest = number_three
    
print("The Largest")

File: dayTwo/java/level two/TaskFive.java
    import java.util.Scanner;

    public class TaskFive {

    public static void main (String [] args) {


    Scanner inputCollector = new Scanner (System.in);

    System.out.print("Enter first number: ");
    int inputOne = inputCollector.nextInt();
    
    System.out.print("Enter second number: ");
    int inputTwo = inputCollector.nextInt();
    
    System.out.print("Enter third number: ");
    int inputThree = inputCollector.nextInt();
    
    int largest = inputOne;
    
        if (inputTwo > largest) {
        largest = inputTwo;
        }
        
        else if (inputThree > largest) {
        largest = inputThree;
        }
        
       System.out.println("Largest Number is: "+ largest);
   
     

    }


    }
